conference.js

```diff
@@ -1820,7 +1820,11 @@ export default {
 
         room.on(
             JitsiConferenceEvents.DOMINANT_SPEAKER_CHANGED,
-            id => APP.store.dispatch(dominantSpeakerChanged(id, room)));
+            id => {
+                APP.store.dispatch(dominantSpeakerChanged(id, room));
+                APP.API.notifyDominantSpeakerChanged(id);
```

Comment:
Can you please move the call to APP.API here? https://github.com/jitsi/jitsi-meet/blob/master/react/features/external-api/middleware.js

Reply:
Hooking onto "DOMINANT_SPEAKER_CHANGED" might be easiest. So somewhere in one of the switch statements:
```
import {
    ....
    DOMINANT_SPEAKER_CHANGED,
    ....
} from '../base/participants';
...
case DOMINANT_SPEAKER_CHANGED:
    APP.API.notifyDominantSpeakerChanged(action.participant.id);
    break;
```

Reply:
Yep, will do. Thanks!!